mysql-test/suite/galera/r/galera_restart_on_unknown_option.result

```diff
@@ -4,6 +4,8 @@ call mtr.add_suppression("The privilege system failed to initialize correctly");
 call mtr.add_suppression("SST failed/interrupted");
 call mtr.add_suppression("IST was interrupted");
 call mtr.add_suppression("IST didn't contain all write sets");
+call mtr.add_suppression("Attempting backtrace");
```

Comment:
I think this one was fixed on 8.0.32 release branch.
@venkatesh-prasad-v

Reply:
Revert this?